From c5eb5474c8f427f3d94281aadec0ecac5c13ebb4 Mon Sep 17 00:00:00 2001
From: thekinhnguyen <the.nguyen.vx@renesas.com>
Date: Tue, 23 Feb 2021 10:59:00 +0700
Subject: [PATCH 069/135] arch: arm64: configs: defconfig: enable
 CONFIG_SPI_SPIBSC_RZG2L

Enable support SPI Multi IO as default.

Signed-off-by: thekinhnguyen <the.nguyen.vx@renesas.com>
---
 arch/arm64/configs/defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index a5390d1..ab2f4a0 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -324,6 +324,7 @@ CONFIG_I2C_UNIPHIER_F=y
 CONFIG_I2C_RCAR=y
 CONFIG_I2C_CROS_EC_TUNNEL=y
 CONFIG_SPI=y
+CONFIG_SPI_SPIBSC_RZG2L=y
 CONFIG_SPI_ARMADA_3700=y
 CONFIG_SPI_BCM2835=m
 CONFIG_SPI_BCM2835AUX=m
-- 
2.7.4

